src/http/params.cr

```diff
@@ -76,6 +76,11 @@ module HTTP
       end
     end
 
+    # Returns the empty `HTTP::Params`.
+    def self.new
+      Params.new({} of String => Array(String))
```

Comment:
Why not simply add a default to `def initialize`, then document that?

Doc should be
> Returns an empty `HTTP::Params`.